.github/workflows/lighthouse-ci.yml

```
@@ -1,5 +1,94 @@
+
+      - name: Generate summary
+        id: gen-summary
+        run: |
+          apt-get update && apt-get install -y jq bc >/dev/null
+          SCORE=$(jq -r '.categories.performance.score * 100' lhreport.json)
+          LCP=$(jq -r '.audits["largest-contentful-paint"].numericValue' lhreport.json)
+          FID=$(jq -r '.audits["max-potential-fid"].numericValue' lhreport.json)
+          CLS=$(jq -r '.audits["cumulative-layout-shift"].numericValue' lhreport.json)
+          cat > lhci-summary.md <<EOF
+## Lighthouse CI summary
+
```

actionlint (1.7.12)

[error] could not parse as YAML: did not find expected key

(syntax-check)